modules/artifact/src/main/java/org/mule/module/artifact/classloader/JdbcResourceReleaser.java

```diff
@@ -264,4 +280,98 @@ private void shutdownMySqlConnectionCleanupThreads(Class<?> classAbandonedConnec
     throw new ClassNotFoundException("No MySql's AbandonedConnectionCleanupThread class was found");
   }
 
+  private void disposeOracleDriverThreads() {
```

Comment:
this method should not finish return until all the interrupted threads are dead, because they may take a while in actually finishing